src/cljs/harja/views/urakka.cljs

```diff
@@ -100,6 +113,18 @@
          ^{:key "toteumat"}
          [toteumat/toteumat ur])
 
+       "Toteumat"
```

Comment:
Pitikö olla toimenpiteet? Ainakin käyttäjälle näkyvässä otsikossa

Näyttäis olevan harja.domain.vesivayla.toimenpide, joten vois muuallakin koodissa puhua vesiväylien toimenpiteistä, ei toteumista.